website/orders/templates/orders/order_admin_list.html

```diff
@@ -296,6 +296,35 @@ <h3 class="mt-3">Orders total</h3>
                     })
                 });
             },
+            order_to_bottom_of_list(order) {
+                if (confirm("Are you sure you want to put this order on the bottom of the list? This can not be undone!")) {
+                    fetch(
+                        `/api/v1/shifts/{{ shift.id }}/orders/${order.id}/`,
+                        {
+                            method: 'PATCH',
+                            headers: {
+                                "X-CSRFToken": get_csrf_token(),
+                                'Content-Type': 'application/json',
+                                "Accept": 'application/json',
+                            },
+                            body: JSON.stringify({
+                                'deprioritize': true,
+                                'prioritize': false
+                            })
+                        }
+                    ).then(response => {
+                        if (response.status === 200) {
+                            return response;
+                        } else {
+                            throw response;
+                        }
+                    }).then(() => {
+                        if (typeof (update_refresh_list) !== 'undefined') {
+                            update_refresh_list();
+                        }
```

Comment:
So below this, add a `tata.success` with a success message (you can also see how this works on the normal order page).